arangod/Aql/IResearchViewNode.cpp

```diff
@@ -1333,19 +1361,18 @@ std::pair<bool, bool> IResearchViewNode::volatility(
 }
 
 void const* IResearchViewNode::getSnapshotKey() const noexcept {
-  if (ServerState::instance()->isDBServer()) {
-    // TODO We want transactional cluster, now it's not
-    // So we don't make sense to make ViewSnapshotView for restrictSources
-    // and we can use node address as key
-    return (_options.restrictSources || _view == nullptr)
-               ? static_cast<void const*>(this)
-               : static_cast<void const*>(_view.get());
-  } else if (ServerState::instance()->isSingleServer()) {
-    return _view.get();
-  }
-
-  TRI_ASSERT(false);
-  return nullptr;
+  // if (ServerState::instance()->isDBServer()) {
```

Comment:
Old code fragements still needed here?

Reply:
We want to restore Snapshot "views" in future PRs. Just now it is broken :( , so we disabled this code temporary